docs/wiki/framework/wardrobe-framework.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: wiki
+title: Wardrobe Framework
+description: Explains the wardrobe system and how to make a compat.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 20
+  patch: 0
+---
+
+
+## 1. Overview
+
+The Wardrobe Addon gives the player the opportunity to change/modify their current wearables into different variants.
+
+For example, if a uniform has a normal variant and a "Rolled-Up Sleeves" variant, the player will be able to use ether an ACE Self Interaction or the CBA Context Menu in the Inventory.
+
+Since there is no reliable, common pattern in terms of class inheritence, not even within the same DLC, each possbile variant has to be defined individually within the classes config properties.
+
+### 1.1 Components
+
+Components are an optional assistance to define a group of variants where a variant is made of one or more components.
+For example, a bandanna with sunglasses would have the the bandanna and the sunglasses as components, while the bandanna and sunglasses themself only have themselves as a component.
+
+If the player switches from a variant with more components to a variant with less components, the surplus components will be deposited in their inventory. Vise Versa, if the player wants to change from a variant with less components to a variant with more components, they are required to have the missing components in their inventory, which will be removed on conversion.
+
+More examples can be found below.
+
+### 1.2 Config Guidelines
+
+- An Container (Uniform, Vest, Backpack) should never change its maxLoad, unless there is a really good reason for it. This will result in the loss of items.
```

Comment:
```suggestion
- A Container (Uniform, Vest, Backpack) should never change its `maxLoad`, unless there is a really good reason for it. This will result in the loss of items.
```